package.json

```diff
@@ -63,6 +63,7 @@
   },
   "dependencies": {
     "@ember/test-waiters": "^2.4.3 || ^3.0.0 || ^4.0.0",
+    "@glimmer/env": "^0.1.7",
```

Comment:
Yet another one in the category of "I don't know how this was working before" 🙈 

Without this dependency declared typescript would complain, but having it as a devDependency you get this error: 

```
ember-a11y-testing tried to import "@glimmer/env" in "ember-a11y-testing/test-support/setup-middleware-reporter.js" from addon code, but "@glimmer/env" is a devDependency. You may need to move it into dependencies.
```

Which is 100% right 🤷 If you're importing a thing in your code then it needs to go in the dependencies. This is another one of those things that `yarn` hides from you because it doesn't setup your deps right 🙈